test/OrchardCore.Tests/Modules/OrchardCore.Resources/SubResourceIntegrityTests.cs

```diff
@@ -6,6 +6,8 @@ namespace OrchardCore.Tests.Modules.OrchardCore.Resources;
 
 public class SubResourceIntegrityTests
 {
+    private static readonly HttpClient _httpClient = new();
```

Comment:
You also need to dispose this. It doesn't actually need to be static, you can just add a `using var` into the test method.

Reply:
> You also need to dispose this

Agree

> It doesn't actually need to be static

The `GetSubResourceIntegrityAsync()` is `static`

Reply:
And you can change that, because you have the infinite power of Visual Studio at your hand :). However, I'm not arguing for that. You can just pass `HttpClient` into the method. The latest implementation of disposing a static field from an instance method is a wrong approach to this.

Reply:
>And you can change that, because you have the infinite power of Visual Studio at your hand

Sure :)

Reply:
I committed what I think is simpler, please check.